### DISCUSSION TITLE
Changing ports From Default

### DISCUSSION THREAD

**Discussion by Whatchawnt** [siemens/edgeshark#55] [Q&A] [answered]
What modifications need to be changed in order to get this working for Systems that have ports 5001, and 5000 already in use? I tried getting this to work on a Synology NAS; which uses ports 5000 and 5001 (for displaying the OS's GUI), by modifying the **ports** in the edgeshark/ghostwire .yml file. But to no avail.

**Comment by thediveo** [comment, marked as answer]
You should be good with a modified docker-compose.yaml where you just change the PORT exposure of the front service "edgeshark" to "5555:5001", keeping the container setup otherwise as is. You don't need to change anything for the backend gostwire service, as it does not expose its port on the host; it's used solely internally.

I'll try to put up an additional "alternative" docker compose file with notes in the next days.

**Reply by thediveo** [reply]
I've pushed an update to the documentation, both README and documentation site. If possible, please use the "combined" docker compose yaml file from `deployments/wget` in the Edgeshark repo, not service-individual composer files. The main difference is, that the combined service deployment file doesn't expose the "backend" G(h)ostwire service separately, because you need to go through the "frontend" capture service in all cases.
